### ISSUE TITLE
Repo README requires using Node version 10.16.3, unsupported by npm or yarn 

### ISSUE THREAD

**Issue by grempe** [stellar/js-stellar-base#494]
The README for developers states that only Node version 10.16.3 shall be used for development and test of the code in this repo.

https://github.com/stellar/js-stellar-base#to-develop-and-test-js-stellar-base-itself

This version of Node is long ago considered outdated and insecure and should not be supported by this library.

It is not possible to install a modern version of `npm` or `yarn` that works with this version of `node`. Version `10.16.3` expects an old version `6.9.0` of `npm`.

## NPM

https://github.com/npm/cli#requirements

```
One of the following versions of Node.js must be installed to run modern npm:

12.x.x >= 12.13.0
14.x.x >= 14.15.0
16.0.0 or higher
```

Trying to install `yarn` with a modern version of `npm` and this ancient version of `node` results in the following error.

```
❯ npm install --global yarn
npm does not support Node.js v10.16.3
You should probably upgrade to a newer version of node as we
can't make any promises that npm will work with this version.
You can find the latest version at https://nodejs.org/
/usr/local/lib/node_modules/npm/lib/npm.js:32
  #unloaded = false
  ^

SyntaxError: Invalid or unexpected token
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at module.exports (/usr/local/lib/node_modules/npm/lib/cli.js:22:15)
    at Object.<anonymous> (/usr/local/lib/node_modules/npm/bin/npm-cli.js:2:25)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
```

Version 10.16.3 of `node` was released in 2019 and is now past its End Of Life (EOL) date and should no longer be used and is no longer supported.

https://nodejs.org/en/about/releases/

https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#10.16.3

NodeJS 12.x will fall out of maintenance status and be EOL on three months and should not be the minimum version either. Node 14.x should be the new minimum supported for development.

**Comment by Shaptic** [comment]
You are one step ahead of me: I had just gotten the ok to bump to v12. I will bump to 14 once it's EOL, in order to smooth the transition from 10.x.

**Comment by Shaptic** [comment]
Hah, even better: I [did this](https://github.com/stellar/js-stellar-sdk/pull/743/files#r789314703) for the js-stellar-sdk package but not here. Docs are fixed by #499.
